src/style.scss

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+
+  @for $row from 1 through 5 {
+    &--#{$row}:nth-child(-n + #{$row}) {
+      background-image: url(images/star-active.svg);
+      background-position: 0;
+    }
+  }
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-size: cover;
+  background-position: center;
+  height: 12px;
+  width: 12px;
+  margin: 2px 6px 2px 2px;
```

Comment:
margin-right between stars is 4px and don't forget to remove margin-right from the last stars in a row, use :last-child rule to do it